app/routing/DeparturesRouter.scala

```diff
@@ -101,4 +101,19 @@ class DeparturesRouter @Inject() (
     case _                                                    => v1Departures.getDeparturesForEori(updatedSince)
   }
 
+  def sendMessageDownstream(departureId: String): Action[Source[ByteString, _]] = route {
```

Comment:
A similar issue here - I'd rather name this after the version 2 action, rather than the version 1 one.

Reply:
done